ci/generate-workflows.py

```diff
@@ -944,7 +944,9 @@ def tag(self):
     Target(container="fedora:36"),
     Target(container="fedora:37"),
     Target(container="fedora:38"),
-    Target(container="alpine:3.15"),
+    Target(container="alpine:3.16"),
+    Target(container="alpine:3.17"),
+    Target(container="alpine:3.18"),
```

Comment:
I only want to build one version for alpine; I don't think anyone other than you uses it and building so many versions clogs up CI, making it less useful for the systems that most people do use.

Which version are we going to go with?

Reply:
I can remove it altogether since I have to build wezterm on different platforms myself (due to incompatibility of mux server) anyway, so I can do it on Alpine as well